Pollo/ViewControllers/PollBuilderViewController.swift

```diff
@@ -304,15 +305,15 @@ class PollBuilderViewController: UIViewController {
             print("cannot start question before viewdidload")
             return
         }
-        
+
         delegate?.showNavigationBar()
         dismiss(animated: true, completion: nil)
         hideKeyboard()
         
         switch questionType { 
         case .multipleChoice:
             answerChoices = mcPollBuilder.getChoices()
-            let question = loadedMCDraft?.text ?? ""
+            let question = mcPollBuilder.questionText ?? ""
```

Comment:
Can you explain this change?

Reply:
For some reason, we were loading the text field from `loadedMCDraft` for a poll which means if the admin did not make a question from a draft, the question field they typed wasn't loaded into the poll. Now if an admin makes a question directly `mcPollBuilder.questionText` loads what they typed in that field and if they use a draft, `mcPollBuilder.questionText` also gets set with what `loadedMCDraft.text`, so it covers both cases.